go.sum

```diff
@@ -1,38 +1,66 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Are these `go.sum` changes required?

Reply:
regenerate  the go.sum with current go.mod,
but I found the go.sum here is different from ee/go.sum, I am not  sure which one is correct. It seems that ee/go.sum has done some unnecessary changes before.